src/superio/nuvoton/nct5104d/superio.c

```diff
@@ -107,41 +107,48 @@ static void route_pins_to_uart(struct device *dev, bool to_uart)
 	pnp_write_config(dev, 0x1c, reg);
 }
 
-static void enable_gpio_io_port(struct device *dev)
+static void enable_gpio_io_port(struct device *dev, bool enable_wdt1)
```

Comment:
no need to pass bool. Simply pass enable_wdt1 option as a u8 parameter. Then you could simply compare the variable to zero like :

```
if (enable_wdt1 == 0)
```